dts/bindings/sensor/st,iis2iclx-common.yaml

```diff
@@ -11,15 +11,19 @@ properties:
         The property value should ensure the flags properly describe
         the signal that is presented to the driver.
 
-    int-pin:
+    drdy-int:
```

Comment:
We need to be careful about this, as it breaks anybody's out-of-tree devicetree file that uses `int-pin` for this property.

Probably best to leave it as is, but if it's going to change both should be present, `int-pin` should be marked deprecated, and the driver should accept both until `int-pin` is removed.

If we don't have that documented as a policy somewhere, we should.  @galak @mbolivar-nordic

Reply:
I saw in #31965 that properties with name changed was marked depracated. I can do the same in fact. Thanks for pointing this out.